core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -722,74 +722,83 @@ package object config {
       .booleanConf
       .createWithDefault(true)
 
-  // Blacklist confs
-  private[spark] val BLACKLIST_ENABLED =
-    ConfigBuilder("spark.blacklist.enabled")
+  private[spark] val EXCLUDE_ON_FAILURE_ENABLED =
+    ConfigBuilder("spark.excludeOnFailure.enabled")
       .version("2.1.0")
```

Comment:
that is a good question, I left it because the config was just renamed but same functionality since 2.1.0.  Also one of the other PRs for renaming slave we left this as well.  
If others have thoughts or we have some standard on this let me know.

Reply:
I guess it depends if you consider the "from version" to represent when the config string was added, or when the functionality was added. I think the former makes more sense, but I certainly wouldn't know if there was an established convention around this.

Reply:
I think we need to update the version. I once renamed a configuration too and @dongjoon-hyun asked to update the version.

Reply:
ok, I guess if that is the case. Like I mentioned one of the other renames didn't change this.   I think it depends on if we are tracking the name of the feature it controls with this?  either way I think what we decide we should document. There is also no documentation on the .version function in the ConfigBuilder so we should really add something there ( I can if we agree up on something).  It ends up showing up on the documentation page under Since Version so I guess looking at that we probably should update it as this rename won't work on older version.